app/models/appointment_summary.rb

```diff
@@ -4,8 +4,9 @@
 
 class AppointmentSummary < ActiveRecord::Base
   DIGITAL_BY_DEFAULT_START_DATE = Date.new(2016, 6, 21)
+  # bassed off: https://github.com/alphagov/notifications-utils/blob/master/notifications_utils/recipients.py#L22
   EMAIL_REGEXP = Regexp.union(
-    /\A[^\@\s]+@[^\@\.\s]+(\.[^\@\.\s]+)+$/,
+    /\A([^";@\s\.]+\.)*[^";@\s\.]+@([^";@\s\.]+\.)+[a-z]{2,10}\z/,
```

Comment:
I think in this case it is fine as we are not storing the email address to create a customer, but to immediately send a notification email (optional). 

The regex is taken from the system that sends the notification so it would break anyway - just later in a background job and then require it to be manually reviewed.

Added to which the email is being entered by a third party - so scope for error.

The use case here is to flag known issues in the email address before the hit the notify system.